src/cmd-run

```diff
@@ -64,6 +66,9 @@ while [ $# -ge 1 ]; do
         -m)
             VM_MEMORY="$2"
             shift 2 ;;
+        --size)
+            VM_DISKSIZE="$2"
```

Comment:
Shouldn't this be e.g. `VM_DISKSIZE="${2}G"`?

Reply:
Good catch, fixed!